app/localization/translated/be.json

```diff
@@ -120,6 +120,7 @@
   "Attachments.noAttachmentsMessage": "Няма ўкладанняў для адлюстравання",
   "AttributeEditor.attributeKeyLengthHint": "Ключ павінен мець памер ад 1 да 512 сімвалаў",
   "AttributeEditor.attributeValueLengthHint": "Значэнне павінна мець памер ад 1 да 512 сімвалаў",
+  "AttributeEditor.attributeFilterValueLengthHint": "Значэнне павінна мець памер не больш за 512 сімвалаў",
```

Comment:
Also I have a question regarding the hints.
They are look pretty similar. Can we leave just one message for all cases?
Point to discuss with BA.

Reply:
I've already discussed it with BA.
And the reason for using such a hint is the following.
If we have the hint 'Value should have the size from 1 to 512 symbols' it automatically makes that field required.
But in this case Value is not required and we only need to restrict users to enter a value longer than 512 symbols.